src/app/[locale]/_component/accordion/faqDefault.tsx

```diff
@@ -73,8 +77,47 @@ const fifthBlockFaqRick = {
     </Link>
   ),
 };
+const fifthRevokeWalletFaqRick = {
+  Link: (chunks: React.ReactNode) => {
+    const { locale } = useParams()
+    return <Link
+      fontWeight={600}
+      href={`/${locale}${ROUTES.PROFILE_BLOCK}`}
+    >
+      {chunks}
+    </Link>
+  },
+};
+
+export const FAQ = ({ flow = Flows.LOGOUT, onOpenFAQ }: FAQProps) => {
```

Comment:
why do we need this code now?

Reply:
As it is currently necessary to monitor the reading of FAQs through custom events, we must implement tracking for when the user opens the accordion. Unfortunately, the accordion component imported from @pagopa/pagopa-editorial-components does not offer a dedicated function for managing such operations. Therefore, we are incorporating an Observable to facilitate tracking when the FAQs are opened